lms/djangoapps/experiments/utils.py

```diff
@@ -104,8 +105,11 @@ def is_enrolled_in_course_run(course_run, enrollment_course_ids):
     """
     Determine if the user is enrolled in this course run
     """
-    course_run_key = CourseKey.from_string(course_run.get('key'))
-    return course_run_key in enrollment_course_ids
+    try:
+        course_run_key = CourseKey.from_string(course_run.get('key'))
+        return course_run_key in enrollment_course_ids
+    except InvalidKeyError:
+        return False  # Invalid course run key. Assume user is not enrolled.
```

Comment:
Emma's musing: if the data is bad, then that's a problem. Maybe instead of swallowing this, we log it or warn about it?